packages/core/src/reporters/ProgressAppendOnlyReporter.ts

```diff
@@ -20,7 +20,7 @@ export default class ProgressAppendOnlyReporter extends ProgressKeeper {
 
   private render() {
     process.stdout.write(
-      `Mutation testing ${this.getPercentDone()} (ETC ${this.getEtc()}) ` +
+      `Mutation testing ${this.getPercentDone()} (elapsed: ${this.getElapsedTime()}) (remaining: ${this.getEtc()}) ` +
```

Comment:
Could we lob them together? `(elapsed: 1m 45sec, remaining: 17h, 16m)`

Reply:
Done for the progress append-only reporter, as suggested here. I think we should do the same thing in the default progress reporter (in the progress bar).